frontend/src/component/ui/SearchBar.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { render, fireEvent, Matcher } from '@testing-library/react';
+import { SearchBar } from './SearchBar';
+
+describe('Searchbar', () => {
+  let changeHandler: () => void;
```

Comment:
Put those inside each test case so that each of them is independent from the other.